src/Donations/Micros/DonationTypes/FundingDonations.tsx

```diff
@@ -104,12 +104,18 @@ function FundingDonations({ setopenCurrencyModal }: Props): ReactElement {
               <div
                 key={index}
                 onClick={() => {
-                  setquantity(option.quantity);
+                  setquantity(
+                    paymentSetup.costIsMonthly && frequency == "yearly"
```

Comment:
Do you know an example project where costIsMonthly is set? If this value is not set our old logic seems to work fine.

Reply:
Ok, found one myself: https://donate-with-planet-jfvtupyne-planetapp.vercel.app/?to=proj_LOxkf5GYI054Fi0HcEUF3dKu